homeassistant/helpers/start.py

```diff
@@ -47,7 +47,9 @@ def cancel() -> None:
         if unsub:
             unsub()
 
-    unsub = hass.bus.async_listen_once(event_type, _matched_event)
+    unsub = hass.bus.async_listen_once(
+        event_type, _matched_event, run_immediately=False
+    )
```

Comment:
Utility meter tests break with this set to True

Will try to remove this in the future as it appears to be related to an open issue https://github.com/home-assistant/core/issues/114908